internal/cmd/environment.go

```diff
@@ -49,6 +129,46 @@ func environmentDumpCmd() *cobra.Command {
 	return &cmd
 }
 
+func printStore(msgData runnerv1.MonitorEnvStoreResponse_Snapshot) error {
+	io := iostreams.System()
+
+	// TODO: Clear terminal screen
+
+	table := tableprinter.New(io.Out, io.IsStdoutTTY(), io.TerminalWidth())
+	table.AddField(strings.ToUpper("Name"))
+	table.AddField(strings.ToUpper("Value"))
+	table.AddField(strings.ToUpper("Spec"))
+	table.AddField(strings.ToUpper("Origin"))
+	table.AddField(strings.ToUpper("Updated"))
+	table.EndRow()
+
+	for _, env := range msgData.Snapshot.Envs {
+		value := env.ResolvedValue
+
+		if env.Status.Number() == runnerv1.MonitorEnvStoreResponseSnapshot_STATUS_MASKED.Number() {
+			value = "[masked]"
+		} else if env.Status.Number() == runnerv1.MonitorEnvStoreResponseSnapshot_STATUS_HIDDEN.Number() {
+			value = "****"
+		}
+
+		table.AddField(env.Name)
+		table.AddField(value)
+		table.AddField(env.Spec)
+		table.AddField(env.Origin)
+
+		t, err := time.Parse(time.RFC3339, env.UpdateTime)
+		if err == nil {
+			table.AddField(t.Format(time.RFC1123Z))
```

Comment:
maybe just `t.Format(time.DateTime)` here? on the client it's okay to show "local time"